### ISSUE TITLE
Icons of blinds do not change

### ISSUE THREAD

**Issue by braindead1** [Zefau/ioBroker.jarvis#10]
Icons of blinds do not change depending on the value as described in the wiki.

The following JSON is used to create the device on the screenshot below.

```
{
   "wohnzimmer_rolladen":{
      "name":"Wohnzimmer",
      "room":"wohnzimmer",
      "function":"blind",
      "states":{
         "level":{
            "state":"hm-rpc.1.XXXXXXXXXX.1.LEVEL",
            "action":"hm-rpc.1.XXXXXXXXXX.1.LEVEL"
         },
         "activity":{
            "state":"hm-rpc.1.XXXXXXXXXX.1.STOP",
            "action":"hm-rpc.1.XXXXXXXXXX.1.STOP"
         }
      },
      "jarvis":{
         "group":"rolladen_erdgeschoss",
         "primary":"level"
      }
   }
}
```

![blind](https://user-images.githubusercontent.com/2136249/76251074-be68e680-6246-11ea-8322-4165dde43805.png)

**Comment by Zefau** [comment]
This is indeed not yet implemented correctly. It's on my list :)

**Comment by Zefau** [comment]
Should be fixed with v0.0.5. Could you please test and verify? Thanks

**Comment by braindead1** [comment]
The bug is fixed in v0.0.5.
